### ISSUE TITLE
[💡 Feature Request]: 

### ISSUE THREAD

**Issue by mpgbur** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#880]
### What's the Problem?

feature request

### What's your solution or idea?

i would like there to be a way to control the computer with the assistant. mainly just turn on turn off etc.

### Alternatives Considered

just a request

### App Version

v1.0.0-beta

### OS Version

windows11

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%E2%9C%A8+enhancement%22+) and didn't find any duplicate issue
- [X] This issue only covers one particular feature request

**Comment by 4g0tt3nSou1** [comment]
Power Control would be rather complicated, Technically, this can already be done _IF_ you have a compatible smart outlet, and I believe there are a few methods of making it so a desktop automatically turns on when it has power (I think there is a trick with either the power switch jumper, and some PSUs have just this as a native capability?). There isn't really anything from the developers point-of-view that could change this, because for power control, while it already can, for example, tell a TV to turn on, it can't do this to a device that doesn't have any hardware method for doing so. Theoretically, you could build a separate "Smart" device to act in place or alongside your computer's power switch too, but really that's on you to implement. 

TL:DR; that's already a feature, technically, with the Google Assistant API, just you have to have a setup that allows hardware control via software/API Calls to do so.

**Comment by 4g0tt3nSou1** [comment]
Other system actions (Stop playing media/Start playing media, Set a timer, Add a reminder, etc) would be possible for other APIs but idk if that's what ur asking.
